File: codekata/mathematics/N_is_a_power_of_K.py
# N is a power of K
# get two input from user in a single line
N,K=list(map(int,input().split()))
# define power function
def power():
# if condition check whether the N is a power of K
 if(N==K*K):
  c="yes"
  return c
 else:
  c="no"
  return c
# call the power function and store it in output
output=power()
# print output value
print(output)

File: codekata/javascript/mathematics/N_is_a_power_of_K.js
// Given 2 numbers N and K.check if N is a power of K.Print 'yes' if it is a power of k otherwise print 'no
const readline=require('readline');
const inp=readline.createInterface({
    input:process.stdin
});
const userInput=[];
inp.on("line",(data)=>{
    userInput.push(data);
});
inp.on("close",()=>{
    var S=userInput[0].split(" ");
    var N=S[0];
    var K=S[1];
 if(N==K*K)
  console.log("yes");
 else
  console.log("no");
  });
